test/testsuppressions.cpp

```diff
@@ -669,7 +669,9 @@ class TestSuppressions : public TestFixture {
 
     void inlinesuppress_unusedFunction() const { // #4210, #4946 - wrong report of "unmatchedSuppression" for "unusedFunction"
         Suppressions suppressions;
-        suppressions.addSuppression(Suppressions::Suppression("unusedFunction", "test.c", 3));
+        auto spr = Suppressions::Suppression("unusedFunction", "test.c", 3);
```

Comment:
generally.. I think it's better to write out the full name. `suppression` is better that `spr` unless it's an abbreviation used everywhere..